src/components/TimeSeries/TimeSeriesForm.tsx

```diff
@@ -0,0 +1,90 @@
+// TODO: Fix eslint errors
+
+/* eslint-disable @typescript-eslint/no-misused-promises */
+import { format, subDays } from 'date-fns';
+import { useEffect, useState } from 'react';
+import { useForm } from 'react-hook-form';
+
+import { stations } from '../../mocks/timeseries/stations.json';
+import timeseries from '../../mocks/timeseries/timeseries.json';
+import {
+  type Station,
+  type TimeSeriesResponse
+} from '../../types/models/timeSeries';
+import './TimeSeriesForm.scss';
+
+interface TimeSeriesFormProps {
+  setChartData: (data: TimeSeriesResponse) => void;
+}
+
+const DATE_FORMAT = 'yyyy-MM-dd';
+
+function TimeSeriesForm({ setChartData }: TimeSeriesFormProps) {
+  const [stationsList, setStationsList] = useState<Station[]>([]);
+  const [isError] = useState<boolean>(false);
+
+  const { register, handleSubmit } = useForm({
+    defaultValues: {
+      dateFrom: format(subDays(new Date(), 7), DATE_FORMAT),
+      dateTo: format(new Date(), DATE_FORMAT),
+      types: ['PM2-5'],
+      stations: ['KRK']
+    }
+  });
+
+  const fetchData = async (formData: FormData): Promise<void> => {
+    setChartData(timeseries);
```

Comment:
```suggestion
    setChartData(timeseries as unknown as TimeSeriesResponse);
```
This imported json structure is correct but TS will not properly recignize union of strings type so you can force it to use correct type.